app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    com, f_name, s_name = command.split(" ")
```

Comment:
The variable names `com`, `f_name`, and `s_name` are not descriptive. According to the checklist, you should use descriptive and correct variable names. Consider renaming them to something like `command_name`, `source_file_name`, and `destination_file_name`.